src/app/modules/form/admin-form/admin-form.service.ts

```diff
@@ -636,6 +637,35 @@ export const updateForm = (
   })
 }
 
+/**
+ * Updates the collaborators of a given form
+ * @param form the form to update collaborators fo
+ * @param updatedCollaborators the new list of collaborators
+ *
+ * @returns ok(collaborators) if form updates successfully
+ * @returns err(PossibleDatabaseError) if any database errors occurs
+ */
+export const updateFormCollaborators = (
+  form: IPopulatedForm,
+  updatedCollaborators: Permission[],
+): ResultAsync<Permission[], PossibleDatabaseError> => {
```

Comment:
this is a good point - i'll add `_id?` to the type of `Permission` and fix the tests accordingly!

Reply:
I think `_id` was omitted in the types because we don't actually use them. I guess there is no harm adding it in as a type anyways and probably adds to the correctness haha